src/librustdoc/html/static/main.js

```diff
@@ -1411,6 +1411,14 @@ function getSearchElement() {
             return tmp;
         }
 
+        // In case someone wants to close the current tag to inject some other unwanted things.
+        // For example: "</span><iframe whatever>"
+        function safeHtml(x) {
+            var e = document.createElement('h1');
+            e.innerHTML = x;
+            return e.innerHTML;
+        }
+
```

Comment:
What prevents someone from injecting nasty tags without closing the span?

Reply:
They can indeed, but it'll be in the documentation as well in any case. It's mostly to prevent breaking the whole HTML rather than "inject" things. My comment might be a bit misleading...

Reply:
Marking as resolved since this code is no longer present in the most recent version.